android/app/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,6 @@
           package="com.templewallet">
 
     <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="android.permission.VIBRATE"/>
```

Comment:
check if this does not break ReactNativeHapticFeedback, please (used when you press on toggle)